### ISSUE TITLE
Add pecl yaml parser for d8

### ISSUE THREAD

**Issue by andypost** [anavarre/drucker#28]
Ref https://www.drupal.org/node/2769555

**Comment by anavarre** [comment]
Thanks, I'll add this very soon.

**Comment by anavarre** [comment]
Done via 2ce24beb0ed9e3a7478d5c9ad51fa17c48450b7c - Released [0.2.8](https://github.com/anavarre/drucker/releases/tag/0.2.8) as well.

```
drucker@drucker:/tmp$ php -c /etc/php/7.0/fpm/php.ini -i | grep yaml
yaml
yaml.decode_binary => 0 => 0
yaml.decode_php => 0 => 0
yaml.decode_timestamp => 0 => 0
yaml.output_canonical => 0 => 0
yaml.output_indent => 2 => 2
yaml.output_width => 80 => 80
OLDPWD => /tmp/yaml-0.1.7
$_SERVER['OLDPWD'] => /tmp/yaml-0.1.7
```
